app/policies/draw_policy.rb

```diff
@@ -58,6 +58,14 @@ def start_lottery?
     edit? && record.pre_lottery?
   end
 
+  def oversubscription?
+    start_lottery?
```

Comment:
Validate that all groups in an oversubscribed size are locked and that we have oversubscription at all.